tests/test_ci_overlap.py

```diff
@@ -1,76 +1,97 @@
 import os
```

Comment:
Don't understand the changes made to the test here, just flagging it for conversation later :)